lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -106,6 +110,7 @@
   overflow-y: auto;
   padding: var(--nx-spacing-6x) var(--nx-spacing-6x);
   white-space: normal;
+  position: relative;
```

Comment:
I'm guessing we no longer need this

Reply:
> I'm guessing we no longer need this

Removed here: [d8f91d2](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/d8f91d2e0612ab9142e22a31b0006587bf94084e)